.storybook/preview.js

```diff
@@ -1,11 +1,13 @@
-import '../src/styles/main.css';
+import '../src/styles/theme.css';
```

Comment:
Oh, should this be _main.css_?  Theme does most of the heavy lifting, but should double check